src/Compilers/CSharp/Portable/Binder/Binder_Operators.cs

```diff
@@ -2137,7 +2137,7 @@ private BoundExpression BindAddressOfExpression(PrefixUnaryExpressionSyntax node
 
         // Basically a port of ExpressionBinder::isFixedExpression, which basically implements spec section 18.3.
```

Comment:
>Basically [](start = 11, length = 9)

This comment is confusing since "is fixed" and "is moveable" are opposites. I suspect the comment was obsolete some time ago, as this probably implements the inverse of `ExpressionBinder::isFixedExpression`. Also, I think the spec section number is now incorrect. It might be better to have a description of what the method does rather than a description of what code it came from.